metro.config.js

```diff
@@ -7,8 +8,27 @@ const gutenbergMetroConfigCopy = {
 gutenbergMetroConfigCopy.resolver.extraNodeModules = new Proxy(
 	{},
 	{
-		get: ( target, name ) =>
-			path.join( process.cwd(), `gutenberg/node_modules/${ name }` ),
+		get: ( target, name ) => {
+			const gutenbergFolder = path.join(
+				process.cwd(),
+				`gutenberg/node_modules/${ name }`
+			);
+			if ( fs.existsSync( gutenbergFolder ) ) {
+				return gutenbergFolder;
+			}
+
+			// let's try find the mobile in the Jetpack submodule. We'll try the .pnpm folder.
```

Comment:
np: Should "mobile" be replaced with "module" in this comment?

Reply:
Oops, my hands see "mobiles" everywhere 😆, thanks for spotting! Fixed with https://github.com/wordpress-mobile/gutenberg-mobile/pull/3575/commits/785bfe2aab960b3e39a085cec235d96c53d09c5d.